### ISSUE TITLE
Skip the step of creating a docs PR in release pipeline and instead check in the code directly

### ISSUE THREAD

**Issue by sima-zhu** [Azure/azure-sdk#2470]
Currently we have a step of creating PR to publish to Doc.MS. We have been reviewing the PR for a while, and now we are confident about the context is stable as what we expected.

In order to release the package owners' burden of looking into each PR. We several options to replace the PR creation steps.
1. Checked in the code directly into master.
2. Checked in the code into some temp branch, and @danieljurek and I are responsible to merge temp branch to master daily or weekly. 

Something to consider:
Since the release pipelines are running concurrently, we need to make sure the commits do not mess up with each others.

**Comment by danieljurek** [comment]
Procedure:

1) Push changes directly to the main branch (today called `master`) 
1) Every day at noon (sufficiently after nightly Docs CI has run against the main branch): 
    1) Checkout a new branch with a static name (e.g. `for-release`) 
    1) Reset the branch to the HEAD of the main branch and push 
    1) Open a PR ([`create-pull-request.yml`](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/pipelines/templates/steps/create-pull-request.yml) might help here) pointing to the `live` branch. The script used by `create-pull-request.yml` will not open a new PR if one is already open 
    1) Dev merges to live if OPS build says there are no errors

**Comment by weshaggard** [comment]
@danieljurek don't you want to make sure this branch's last commit is from the Docs CI?

**Comment by sima-zhu** [comment]
We currently pending on the https://github.com/Azure/azure-sdk-tools/pull/1534. 
Java/JS pipeline: https://dev.azure.com/azure-sdk/internal/_build?pipelineNameFilter=sima

**Comment by danieljurek** [comment]
Python -- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=835683&view=results

**Comment by sima-zhu** [comment]
Enabled the Java and JS daily build right before the Docs Job CI.
